src/Service.Tests/DatabaseSchema-PostgreSql.sql

```diff
@@ -336,6 +345,7 @@ INSERT INTO journals(id, journalname, color, ownername) VALUES (1, 'Journal1', '
 
 INSERT INTO aow("NoteNum", "DetailAssessmentAndPlanning", "WagingWar", "StrategicAttack") VALUES (1, 'chapter one notes: ', 'chapter two notes: ', 'chapter three notes: ');
 INSERT INTO sales(id, item_name, subtotal, tax) VALUES (1, 'Watch', 249.00, 20.59), (2, 'Montior', 120.50, 11.12);
+INSERT INTO books_sold(id,book_name,last_sold_on) values(1, 'Awesome Book', CURRENT_DATE);
```

Comment:
Should MS/MY/PG sql all be consisten and use a literal date versus a shortcut like `CURRENT_DATE`?

Reply:
yeah, makes sense. updated.

Reply:
*updated for table creation when setting default value for last_sold_on